### PR TITLE
Created App class at root of project

### PR DESCRIPTION
**Pull request by vfaltings**
I created an App class at the root of the project so that we can keep global state in the app.

There was an old class that previously extended Application, I'm not sure where that code is used so for the moment I just copied that code into the new App class (this should make it so nothing breaks). I also updated the Manifest to use the new App class

### CODE REVIEW

**Comment by vfaltings** [comment]
I had a look around in the code and the old UserDatabase class that used to extend Application had some code that wasn't used anywhere. So I deleted that class, it's tests and the code associated (didn't seem useful to me)

File: app/src/main/java/com/sdp/swiftwallet/SwiftWalletApp.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.sdp.swiftwallet;
+
+import android.app.Application;
+
+public class SwiftWalletApp extends Application {
```
Comment (0xdieman) [inline app/src/main/java/com/sdp/swiftwallet/SwiftWalletApp.java:5]: Nice app name

**Review by 0xdieman — APPROVED** [review]
Not much to comment, nice PR, GG

**Review by ahominal — APPROVED** [review]
seems good for me, it builds correctly!

**Pull request merged**